src/NativeChargebeeReactNative.ts

```diff
@@ -3,6 +3,7 @@ import { TurboModuleRegistry } from 'react-native';
 
 export interface Spec extends TurboModule {
   configure(site: string, publishableApiKey: string, sdkKey: string): void;
+  retrieveProductIdentifiers(queryParams: Object): Promise<Array<string>>;
```

Comment:
Can we type `queryParams` more specifically?

Reply:
Also, should this be an optional param with default value? 🤔

Reply:
@cb-sriramthiagarajan I was following the listed argument types for conversion between JS and Kotlin/iOS byt following [this](https://reactnative.dev/docs/native-modules-android.html#argument-types) and [this](https://reactnative.dev/docs/native-modules-ios.html#argument-types)
For passing in queryParams which is passed in as key-value/dictionary, the recommended type seems to `Object`. 
Let me try if subtyping works.